include/library_dumper.h

```diff
@@ -77,8 +94,8 @@ class LibraryDumper
 
 
  protected:
-  kiwix::Library* library;
-  kiwix::NameMapper* nameMapper;
+  kiwix::Library* const library;
+  kiwix::NameMapper* const nameMapper;
```

Comment:
`const` was added in the wrong place. I intended `const Library* library;` and `const NameMapper* nameMapper;` (we don't want `LibraryDumper` to be able to call non-const operations on library and/or name mapper objects). Though the immutability of those data-members doesn't hurt either. So let's end up with

```
const kiwix::Library* const library;
const kiwix::NameMapper* const nameMapper;
```

Reply:
Apologies :)
I always mess up between `pointer to const` and `const pointer` :P